### ISSUE TITLE
non-hosted fields inside of `braintree-hosted-fields-wrapper` can't receive keyboard focus

### ISSUE THREAD

**Issue by evan-007** [nathanstitt/react-braintree-fields#18]
Hi,

If you add a non-hosted field to the `demo.jsx` like the `Full name` input here:

```jsx
Date:
<HostedField type="expirationDate" />
Month:
<HostedField type="expirationMonth" />
Year:
<HostedField type="expirationYear" />
Full name:
<input type="text />
CVV:
<HostedField type="cvv" placeholder="CVV" ref={cvvField => { this.cvvField = cvvField; }} />
Zip:
<HostedField type="postalCode" />
```

that input will not be able to receive focus from tabbing through the fields. I think it might be related to how the `Braintree` component [manages focus](https://github.com/nathanstitt/react-braintree-fields/blob/master/src/field.jsx#L28) of the `HostedField`, but I'm not sure. Is there a way to make that element focusable while still having it nested inside of `Braintree` component?

update: this appears to only be an issue when the non-hosted field is between two sibling `HostedField`, it receives focus correctly if it is the last element in the `Braintree` container.

demo: http://watery-plough.surge.sh/ (tab through the fields and you can't focus on the first `Full name` input, but you can on the 2nd one.

demo source: https://github.com/evan-007/braintree-demo/blob/e058803e97549e7152e2e8ecfd54dc7eb7555e88/src/Braintree.js#L91-L117

**Comment by nathanstitt** [comment]
Wow, this is one that I haven't encountered before.  It does appear to be something caused by `braintree-hosted-fields-wrapper` since I've been able to place fields directly into the example Braintree-hosted fields example without issues: https://codepen.io/nathanstitt/pen/gNRLXQ

I don't think it's the focus method you pointed out though, since that's not normally called, it's intended to be called by consumers of the library that need to programmatically focus a field, not by the normal tab flow.

I'll attempt to investigate further and check back later

**Comment by nathanstitt** [comment]
Good news, while I didn't quite figure out the cause of this, it no longer appears to occur with the latest Braintree library.

I just published version `1.3.1` with that as a dependency.  If you're still having issues please give that version a try.

Thanks for reporting the issue
